files/jsondata/L10n-SVG.json

```
@@ -463,7 +486,8 @@
     "ja": "変換関数属性",
     "pt-BR": "Atributos de função de transferência",
     "ru": "Функция передачи атрибутов",
-    "zh-CN": "变换函数属性"
+    "zh-CN": "变换函数属性",
+    "zh-TW": "傳遞函式屬性"
```

There are more results for "轉移函數" / "轉換函數"?

---

Or `遞移`?

see https://zh.wikipedia.org/zh-tw/%E4%BC%A0%E9%80%92%E5%87%BD%E6%95%B0

```suggestion
    "zh-TW": "遞移函式屬性"
```

---

I think that one is fine, too.